README.adoc

```diff
@@ -153,6 +153,11 @@ $ make clean
 
 Here's how to run all available tests. All tests will check all Go packages
 except those in the `vendor/` directory.
+Make sure you have docker and docker-compose available (Probably you may consider adding your user into `docker` group using `usermod -aG docker $USER`).
```

Comment:
Thanks @aslakknutsen for pointing out this thing. Agreed. Without this content in the README is rest of changes okay to go ?

@kbsingh Thanks, i was not aware of it. Actually I did not tried on it. And in future, I will provide related links for such Setup tips (because user can be on another system).